_includes/header.html

```diff
@@ -33,7 +33,7 @@
                                 <a class="list_link" href="/newsroom/">Newsroom</a>
                             </li>
                             <li class="list_item list-expanding_list-item">
-                                <a class="list_link" href="/contact-us/">Contact Us</a>
+                                <a class="list_link" href="#">TBD Content</a>
```

Comment:
Perhaps just throw in some inline style to disable the hover-state? Right now it's not like the other TBAs in that it has the same style as the items around it and isn't greyed out.